src/release.sh

```diff
@@ -29,6 +29,12 @@ kustomize_version=$(curl -sS https://api.github.com/repos/kubernetes-sigs/kustom
   | sed -e 's#^kustomize/##' -e '/-pre/d' \
   | head -n 1)
 
+imagecleanup_version=$(curl -sS https://api.github.com/repos/appuio/image-cleanup/releases \
+  | jq --raw-output \
+      '.[]| select(.prerelease|not) | .tag_name' \
+  | sed '/v3\./d' \
```

Comment:
I guess that's a copy/paste leftover. You can remove the `sed` line, there should be no versions to be excluded.

Reply:
OK, I have removed it.  Sorry, not sure why I copied that part.